pkg/registry/apis/provisioning/jobs/exporter.go

```diff
@@ -39,13 +42,21 @@ func NewExporter(
 ) (Exporter, error) {
 	dynamicClient := parser.Client()
 	folders := dynamicClient.Resource(schema.GroupVersionResource{
-		Group:    "folder.grafana.app",
-		Version:  "v0alpha1",
-		Resource: "folders",
+		Group:    folders.GROUP,
+		Version:  folders.VERSION,
+		Resource: folders.RESOURCE,
 	})
+
+	dashboards := dynamicClient.Resource(schema.GroupVersionResource{
+		Group:    "dashboard.grafana.app",
+		Version:  "v1alpha1",
+		Resource: "dashboards",
+	})
```

Comment:
At this point, I wonder if having a function for specifically common resource clients would be handy... we use them everywhere

Reply:
yes, it's a mess. It's one of the things I have captured in my local notes and I will put our Prioritized List :)